runelite-client/src/main/scripts/OptionsPanelZoomMouseListener.rs2asm

```diff
@@ -80,5 +80,9 @@ LABEL5:
    istore                 4
    iload                  4
    iload                  3
+   load_string            "panelZoomStart"
+   runelite_callback
    invoke                 42
+   load_string            "panelZoomEnd"
```

Comment:
This could be done in Java by setting a bool when you hit `panelZoomStart` then setting it to false when ZoomHandler runs. Not sure if that's actually any better though.